commons/src/main/java/org/infinispan/commons/dataconversion/StandardConversions.java

```diff
@@ -168,6 +170,28 @@ public static Object convertOctetStreamToJava(byte[] source, MediaType destinati
       return marshaller.objectToByteBuffer(source);
    }
 
+   /**
+    * Converts a java object to a sequence of bytes using a ProtoStream {@link SerializationContext}.
+    *
+    * @param source source the java object to convert.
+    * @param sourceMediaType the MediaType matching application/x-application-object describing the source.
+    * @return byte[] representation of the java object.
+    * @throws EncodingException if the sourceMediaType is not a application/x-java-object or if the conversion is
+    * not supported.
+    */
+   public static byte[] convertJavaToOctetStream(Object source, MediaType sourceMediaType, SerializationContext ctx) throws IOException, InterruptedException {
```

Comment:
This in fact converts Java to Protostream format, so it should be called ```converObjectToProtostream```?

Reply:
Sure, but I'll change it to `convertJavaToProtoStream` as that's more consistent with the other method names.